etc/rc.d/rc.local

```diff
@@ -161,7 +161,7 @@ else
     log "Installing /boot/extra packages"
     ( export -f log; find /boot/extra -maxdepth 1 -type f -exec sh -c 'upgradepkg --terse --install-new "$1" | log' -- "{}" \; )
   fi
-  PRIORITY_PLUGINS=("unraid.patch.plg" "dynamix.unraid.net.plg" "dynamix.unraid.net.staging.plg")
+  PRIORITY_PLUGINS=("dynamix.unraid.net.plg" "dynamix.unraid.net.staging.plg")
```

Comment:
Please remove "dynamix.unraid.net.staging.plg" from this list as well, leaving just "dynamix.unraid.net.plg"